Sources/StytchCore/StytchClient/StytchClient.swift

```diff
@@ -21,10 +21,14 @@ public struct StytchClient: StytchClientType {
 
     static let appSessionId: String = UUID().uuidString
 
-    public static var bootStrapData: BootstrapResponseData? {
+    static var bootStrapData: BootstrapResponseData? {
         Self.instance.localStorage.bootstrapData
     }
 
+    public static var disableSdkWatermark: Bool {
+        bootStrapData?.disableSdkWatermark ?? true
```

Comment:
I think we wanna default to false here

Reply:
I went with true in case someone had paid to turn it off and something went wrong and in that case it would show, I figure let some folk get the hidden watermark for free, rather than the ones you paid not get it